crates/shared/src/bad_token/trace_call.rs

```diff
@@ -82,6 +82,7 @@ impl TraceCallDetector {
                 }
             });
         }
+        tracing::debug!("Candidates: {:?}", &candidates);
```

Comment:
If we want to keep this log maybe add the token too so it's more useful.

Reply:
Good catch this was just for internal debugging, will remove